core/runtime.jvm/src/kotlin/Generated.kt

```diff
@@ -0,0 +1,8 @@
+package kotlin
+
+/**
+ * This annotation is present for methods generated by the Kotlin compiler
+ */
+@Retention(AnnotationRetention.RUNTIME)
+@Target(AnnotationTarget.FUNCTION)
+annotation class Generated
```

Comment:
Why is this annotation is a part of `runtime.jvm`? And if it supposed to be Jvm only, according to existing naming convention it should be `JvmGenerated` instead

Reply:
This new annotation is consistent with `@kotlin.Metadata`. You may find it together in the same package.

Reply:
Huh, you right, interesting.

Reply:
@goodwinnk I agree with your comment, but I'm not sure what the implications are with respect to what you expect @andreyfomenkov to change in this PR.